### DISCUSSION TITLE
Display session name not number?

### DISCUSSION THREAD

**Discussion by axolotl-logic** [catppuccin/tmux#364] [Help] [answered]
Hello,

Instead of showing the session number,  I would love to show the same name instead. Same thing with the window name.

The bests from bestie.

**Comment by vdbe** [comment]
Session name is already displayed in the session status module, (default name is the session number).

For window name 

```
set -g @catppuccin_window_current_text "#{window_name}"
set -g @catppuccin_window_default_text "#{window_name}"
```

or 

```
set -g @catppuccin_window_current_text "#W"
set -g @catppuccin_window_default_text "#W"
```

**Comment by axolotl-logic** [comment]
Exceedingly strange, I've renamed the session and can confirm with :ls, but only a number is shown. Using the default config.








---- On Mon, 07 Oct 2024 11:18:38 -0400 ***@***.*** wrote ----



Session name is already displayed in the session status module, (default name is the session number).

For window name

set -g @catppuccin_window_current_text "#{window_name}"
set -g @catppuccin_window_default_text "#{window_name}"


or

set -g @catppuccin_window_current_text "#W"
set -g @catppuccin_window_default_text "#W"

**Reply by vdbe** [reply]
can you show the output of `tmux show -g status-right`?

**Comment by axolotl-logic** [comment]
```
status-right "#{E:@catppuccin_status_user}#{E:@catppuccin_status_directory}"
```

```                                                                                  
status-left ''
```

```
set-option -sg escape-time 10
set -g default-terminal "screen-256color"
set-option -a terminal-features 'xterm-256color:RGB'
set-option -g focus-events on

# Configure the catppuccin plugin
set -g @catppuccin_flavor "macchiato"
set -g @catppuccin_window_status_style "rounded"
# leave this unset to let applications set the window title
set -g @catppuccin_window_default_text " #W"
set -g @catppuccin_window_current_text " #W"
set -g @catppuccin_window_status "icon"
set -g @catppuccin_window_current_background "#{@thm_mauve}"

# Load catppuccin
run ~/.config/tmux/plugins/catppuccin/tmux/catppuccin.tmux

# Make the status line pretty and add some modules
set -g status-left ""
set -g status-right "#{E:@catppuccin_status_user}"
set -ag status-right "#{E:@catppuccin_status_directory}"
```

and :ls shows

```
Dojo: 2 windows (created Mon Oct  7 10:22:01 2024) (attached)
```

The session name is meant to bee "Dojo"

Also, as a side question so I don't open another post later, is it possible to show the git branch of the current directory?

**Reply by vdbe** [reply, marked as answer]
Add `#{E:@catppuccin_status_sesssion}` to right you are currently not displaying the session name anywhere.
As for git I suggest the [gitmux](https://github.com/catppuccin/tmux/tree/main/status#gitmux-module) module or `#(git branch --show-current)`

**Reply by axolotl-logic** [reply]
In terms of the gitmux, is there a way to have the same look and feel as the other modules in catppuccin, i.e. the half pill background color behind the icon?

**Reply by vdbe** [reply]
Yes it's a module like all the other modules

**Reply by axolotl-logic** [reply]
It displays with the icon but without the round/flat pull background color differentiating from the other background color.

**Reply by vdbe** [reply]
Sounds like a bug somewhere but would not know where

**Reply by kjnsn** [reply]
Have you set the colour for it?
